Multisig/Data/Services/Utils/HTTPClient.swift

```diff
@@ -4,61 +4,95 @@
 
 import Foundation
 
-public protocol HTTPRequest {
-
+protocol HTTPRequest {
     var httpMethod: String { get }
     var urlPath: String { get }
     var query: String? { get }
     var body: Data? { get }
     var headers: [String: String] { get }
 }
 
-public extension HTTPRequest {
-
+extension HTTPRequest {
     var query: String? { return nil }
     var body: Data? { return nil }
     var headers: [String: String] { return [:] }
-
 }
 
 /// Synchronous http client
-public class HTTPClient {
-    /// Client error
+class HTTPClient {
```

Comment:
maybe move the error handling out in another PR/ or another issue? This will make this PR more focused

Reply:
Yeah, I agree that there is a room for two PRs :( But I would not like to do it now.